@xen-orchestra/web/src/components/host/system/HostSystemHardwareSpecifications.vue

```diff
@@ -41,5 +46,26 @@ const { host } = defineProps<{
 
 const { t } = useI18n()
 
-const pGpusIds = computed(() => host.PGPUs.join(', '))
+const { get: getPci, isReady: isPciReady } = usePciStore().subscribe()
+const { get: getGpu, isReady: isPgpuReady } = usePGpu().subscribe()
+
+const pGpusName = useArrayReduce(
```

Comment:
```suggestion
const devicesNames = useArrayReduce(
```

Naming ? In lite we got `PciDevicesNames` , we should choose something that's the same on both sides